tests/QueryLanguage/Processor/Doctrine/ORM/ProcessorTest.php

```diff
@@ -106,6 +108,50 @@ public function testPageSizeOptionShouldWork(array $params): void
         self::assertCount(3, $result);
     }
 
+    public function provideParamsForDefaultOrder(): iterable
+    {
+        yield [false, true, '$order(name)'];
+        yield [false, true, 'name'];
+        yield [false, true, 'name, desc'];
+        yield [true, false, '$eq(name)'];
+        yield [false, false, '$order(nonexistent, asc)'];
+    }
+
+    /**
+     * @dataProvider provideParamsForDefaultOrder
+     */
+    public function testShouldOrderByDefaultField(bool $exception, bool $valid, string $defaultOrder): void
```

Comment:
I would prefer to have these tests separated instead a single. 
Something like `testOrderByDefaultFieldShouldThrowOnInvalidOptions` and `testOrderByDefaultFieldShouldWork`

Reply:
Done